### PR TITLE
Bugfix/datepicker

### LINKED ISSUE #1601: Datepicker width issue in internet explorer with react
If datepicker is opened, the width changes so the window covers the whole screen.

Bug was explained in detail with @NoahLiechti

### PR DESCRIPTION
**Pull request by danyball**
Fixes #1601

# Done is when (DoD):
- [x] I have added UI/Unit tests for my changes
- [ ] Old tests and eslint rule are not broken
- [ ] I added modified component properties to the typescript typings
- [ ] Design has been reviewed with (here name of reviewer)
- [x] My changes are accordingly to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)
- [ ] My changes are well documented in the README and showcased in the stories 
- [x] My changes are shortly added to CHANGELOG.md with the issue number
- [x] Tested with IE

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas (focus on **WHY**)
- [ ] I have made/updated the ChangeLog Section in the README

### CODE REVIEW

**Review by raphaellueckl — DISMISSED** [review]
lgtm

File: src/components/20-molecules/datepicker/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 5.0.1
+- We do not set height to component anymore because that causes a bad behavior if you has set a label text
+- Set a width to popup wrapper because of IE (Issue #1601)
```
Comment (markus-walther) [inline src/components/20-molecules/datepicker/CHANGELOG.md]: Wording: Changed implementation internals for `width` due to IE bug (#1601).

File: src/components/20-molecules/datepicker/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 5.0.1
+- We do not set height to component anymore because that causes a bad behavior if you has set a label text
```
Comment (markus-walther) [inline src/components/20-molecules/datepicker/CHANGELOG.md]: Wording: component `height` implementation internals changed.

**Review by markus-walther — CHANGES_REQUESTED** [review]
Looks good to me, 2 minor doc changes as discussed.

**Pull request merged**